COMPATIBILITY.md

```diff
@@ -0,0 +1,40 @@
+# Instrumented Library Compatibility
+
+Instrumentation is provided for the following Go packages.
```

Comment:
```suggestion
Instrumentation is provided for the following Go libraries.
```

Why? Sometimes a library (module) consists of several packages (and even modules?). I think that "library" is a more general term and package and module are implementation specific. Also in OTel terminology we have "instrumented libraries". Related PR https://github.com/open-telemetry/opentelemetry.io/pull/4727